src/wmnds/patterns/content-cards/buttons/_buttons.scss

```diff
@@ -0,0 +1,8 @@
+.wmnds-content-card {
+  &__list {
+    .wmnds-btn {
+      width: 100%;
+      text-align: left;
+    }
+  }
+}
```

Comment:
Could all of this go in the same scss file as the other content card styling?
Just thinking if something changes in the core/button but gets forgotten about here.